addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1068,6 +1072,19 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
     setVisibilityDependency(previewMode, widgetValue, params) {
         this._setVisibilityDependency(widgetValue);
     },
+    /**
+     * @override
+     */
+    async selectDataAttribute(previewMode, widgetValue, params) {
+        await this._super(...arguments);
+        if (params.attributeName === 'maxFilesNumber') {
+            if (params.activeValue > 1) {
+                this.$target[0].setAttribute('multiple', '1');
+            } else {
+                this.$target[0].removeAttribute('multiple');
+            }
```

Comment:
Is it necessary to put '1' as the value of the multiple attribute. I think that having the multiple attribute is sufficient. You can use toggleAttribute so :+1:

Reply:
In JS it is not necessary indeed, but in the XML template, I need to set `t-att-multiple` to 1 because it doesn't work with an empty string :grimacing: 
So for the sake of consistency, shouldn't we keep the 1 here too ? 
Thanks for `toggleAttribute` by the way, I was not aware of it :)

Reply:
Can't you do something like that in XML ? :thinking: 
`t-att="field.maxFilesNumber > 1 and {'multiple': ''} or {}"`